### PR TITLE
252 Add raw chem download to Get Data page

### PR DESCRIPTION
**Pull request by mwfrost**
The link downloads a csv file containing all the RawChem records with no link to the DSSToxLookup table. The ID is included, so users can curate chemicals offline and then add them to the database via SQL tools.

### CODE REVIEW

**Comment by mwfrost** [comment]
The `tag_submit` test is failing, but that is fixed on a different branch

File: dashboard/views/get_data.py [hunk 1]
```diff
@@ -162,9 +162,19 @@ def upload_dtxsid_csv(request):
...
+
+def download_raw_chems(stats):
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="uncurated_chemicals_%s.csv"' % (datetime.datetime.now().strftime("%Y%m%d"))
+
+    writer = csv.writer(response)
+    writer.writerow(['dashboard_rawchem_id',  'raw_cas', 'raw_chem_name', 'rid'])
+    for rawchem in RawChem.objects.filter(dsstox_id=None):
+        writer.writerow([rawchem.id, rawchem.raw_cas, rawchem.raw_chem_name, rawchem.sid if rawchem.sid else '' ])
+
```
Comment (debboutr) [inline dashboard/views/get_data.py]: it should be `rawchem.rid` not `rawchem.sid` up here right?

Comment (dmlyons2) [reply dashboard/views/get_data.py]: @kdionisio what are you looking for in the column headers for the csv for this ticket? is it `raw_chem_id`, `raw_cas`, `raw_chem_name`, `rid`?

Comment (dmlyons2) [reply dashboard/views/get_data.py]: The ticket says " download a .csv file which includes any raw_cas and raw_chem_names which do not yet have an SID assigned" so I am thinking that sid is correct here @debboutr

Comment (mwfrost) [reply dashboard/views/get_data.py]: No, the file needs to show the RID for the users' benefit.

Comment (zachmunzenrider) [reply dashboard/views/get_data.py]: Maybe we could come up with some _less_ descriptive column names to clear up the confusion! ;-)

Comment (mwfrost) [reply dashboard/views/get_data.py]: Good mnemonic: "R is for 'raw'"

**Pull request merged**